pkg/codegen/nodejs/gen.go

```diff
@@ -38,8 +38,8 @@ import (
 	"github.com/pulumi/pulumi/sdk/v2/go/common/util/contract"
 )
 
-// Match k8s version suffix. Examples include "/v1beta1" and "/v1alpha2".
-var k8sVersionSuffix = regexp.MustCompile(`/v\d+((alpha|beta)\d+)?$`)
+// Match k8s version suffix. Examples include "v1beta1" and "v1alpha2".
+var k8sVersionSuffix = regexp.MustCompile(`v\d+((alpha|beta)\d+)?$`)
```

Comment:
Let's leave the `/` in there so we're sure we're picking up a path component rather than something in the middle of a string.

```suggestion
var k8sVersionSuffix = regexp.MustCompile(`/(v\d+((alpha|beta)\d+)?)$`)
```